backend/src/main/java/de/jankahle/capstone/model/Recipe.java

```diff
@@ -0,0 +1,24 @@
+package de.jankahle.capstone.model;
+
+import lombok.*;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
```

Comment:
depending on in which context recipe is later used you may want to exclude builder annotation as you provide a public method with builder to create DBrecipe